frontend/src/utilities/clusterUtils.ts

```diff
@@ -7,7 +7,8 @@ export const getOpenShiftConsoleServerURL = (apiURL?: string): string | null =>
   const { hostname, protocol, port } = window.location;
 
   if (DEV_MODE && apiURL) {
-    let apiURLWithoutPrefix = apiURL.slice('https://api.'.length);
+    let apiURLWithoutPrefix =
+      process.env.CONSOLE_LINK_DOMAIN || apiURL.slice('https://api.'.length);
```

Comment:
Can you move it to https://github.com/opendatahub-io/odh-dashboard/blob/main/frontend/src/utilities/const.ts? All the frontend env const are processed there and re-export to be used by other files.